wp-content/themes/core/components/search_form/css/search-form.pcss

```diff
@@ -48,21 +71,15 @@
 	&:focus {
 		color: var(--color-primary);
 	}
-}
 
-.c-search__submit-button {
-	right: 12px;
+	/* CASE: Search Form in the site header */
+	.c-search--site-header & {
+		right: auto;
+		left: 0;
 
-	&:before {
-		content: var(--icon-search);
-	}
-}
-
-.c-search__clear-button {
-	right: 44px;
-	color: var(--color-neutral-50);
-
-	&:before {
-		content: var(--icon-close);
+		&:before {
+			font-size: 32px;
```

Comment:
Same as above: any possibility we can align with font-sizes and line heights already defined?

Reply:
Similar to the above comment, this is such a one-off usage it's safe to use a pixel value here rather than an out-of-box variable as this won't ever align with our body or heading font values.